src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+.groups > .block:not(:last-child) {
```

Comment:
won't it work like this?
```suggestion
.block:not(:last-child) {
```